hack/errata-check.sh

```diff
@@ -229,7 +229,8 @@ function list-commits {
     local jira_status=""
     local attached=""
     commit_hash=$(echo "$commit" | awk '{ print $1 }')
-    pr_num=$(echo "$commit" | awk '{ print substr($5,2) }')
+    commit_date=$(echo "$commit" | awk -F" " '{ print $2,$3,$4 }')
```

Comment:
This could be made a `local` variable as all the others are

Reply:
Updated.